### PR TITLE
Update FAQ and Troubleshooting

### PR DESCRIPTION
**Pull request by xamionex**
- General MD Rules
- Removing Issues from FAQ and linking people to go to troubleshooting instead
- Fixing general issues with formatting
- More Troubleshooting issues

### CODE REVIEW

**Comment by xamionex** [comment]
You can check my changes [here](https://amione.gitbook.io/northstarwiki)

File: docs/faq.md [hunk 1]
```diff
@@ -42,28 +44,19 @@ and instructions to host a _listen server_:
 [basic-listen-server.md](hosting-a-server-with-northstar/basic-listen-server.md)
 {% endcontent-ref %}
 
-### Q: My main menu screen is blank or only cinematics show up!
-
-A: Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.`, are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods), and included in the release zip).\
-Try deleting `enabledmods.json` as well. Otherwise pay attention in console for your errors if you know what you're doing.
-
```
Comment (GeckoEidechse) [inline docs/faq.md:49]: Is there a reason this was removed from FAQ but not moved to Troubleshooting?

Comment (xamionex) [reply docs/faq.md:49]: I have forgot. Sorry. If you don't mind just copying it to the troubleshooting page cause I'm in school right now

File: docs/installing-northstar/troubleshooting.md [hunk 2]
```diff
@@ -1,28 +1,42 @@
 # Troubleshooting
 
-Generally try to first launch the vanilla game (i.e. not Northstar) if you encounter any issue and see if it also occurs there as well. Some problems can occur when the vanilla game was never launched before using Northstar. 
+Generally try to first launch the vanilla game (i.e. not Northstar) if you encounter any issue and see if it also occurs there as well. Some problems can occur when the vanilla game was never launched before using Northstar.
 
 A lot of problems around the game failing to communicate with Origin can also be prevented by launching Origin before Northstar should you encounter any issues in that regard.
 
-Also note that some solutions described below can also apply to the base game, like issues caused by 10th+ gen Intel CPUs.
+Also note that some solutions described below can also apply to the base game, like issues caused by [10th+ gen Intel CPUs](#cant-join-servers-issues-with-10th-gen-intel-cpus).
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]: Link didn't work for me when testing on [your gitbook mirror](https://amione.gitbook.io/northstarwiki/installing-northstar/troubleshooting).

File: docs/faq.md (hunk 1 above)
Comment (xamionex) [reply docs/faq.md:49]: I don't know if i can do it on phone. Will try

**Comment by GeckoEidechse** [comment]
Actually looks like I just missed it, [the section is there](https://github.com/R2Northstar/NorthstarWiki/pull/38/files#diff-7accce06a795c91ffe8423856d4eb562a44ae7956683739448d9273b4de1c7f2R118), I just `CTRL+F`ed for the wrong keyword. ^^

File: docs/installing-northstar/troubleshooting.md [hunk 3]
```diff
@@ -73,6 +90,55 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
...
+## The Main Menu is blank
+
+- Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.` / are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods) / included in the release zip).\
+- Try deleting `enabledmods.json` inside the R2Northstar folder as well.
+- Otherwise pay attention in console for your errors if you know what you're doing.
+
+## Adding Launch Options
+
+Adding -northstar will start Northstar when launching from your launcher\
+Adding -vanilla will start the normal game when you have Northstar installed
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
Adding `-northstar` will start Northstar when launching from your launcher\
Adding `-vanilla` will start the normal game when you have Northstar installed
```

Just a bit of formatting ;)

**Comment by xamionex** [comment]
Oh right. I totally forgot I did actually add it

**Review by GeckoEidechse — COMMENTED** [review]
Aight overall looks good. Change/fix the two things from review when you get to it and I'll merge it ^^

Also, many thanks for the gitbook mirror of your PR, made reviewing and checking whether things actually work a lot easier <3

File: docs/installing-northstar/troubleshooting.md [hunk 4]
```diff
@@ -73,6 +90,55 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
...
+If your error says `Access Violation | Attempted to read from 0x00000000` specifically DO NOT post just THAT.\
+The real error is most likely slightly higher. Please post that in issues or the discord
+
+## I can't play the Campaign
+
+Campaign is not supported in 1.4, but you could do one of these **"_fixes_"**
+
+1. Doing `ns_auth_allow_insecure 1` in the console
+    - [If you can't open the console](#i-cant-open-the-console)
+2. Adding -vanilla to your launch options
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
2. Adding `-vanilla` to your launch options
```

**Comment by xamionex** [comment]
Heh no problem! Glad you like it.

File: docs/installing-northstar/troubleshooting.md [hunk 5]
```diff
@@ -73,6 +90,55 @@ If you're on Linux, you can set the appropriate environment variable via `env OP
...
+    - [How to add launch options](#adding-launch-options)
```
Comment (xamionex) [inline docs/installing-northstar/troubleshooting.md:144]: Tested on mobile, works

**Review by xamionex — COMMENTED** [review]
I approve myself.

**Review by GeckoEidechse — APPROVED** [review]

**Pull request merged**